### PR TITLE
Fix Snap.atan2

### LINKED ISSUE #507: Snap.atan2 passes only one number to Math.atan2
I'm a complete Snap newbie, so feel free to bite.  :-)   I think there may be an issue with Snap.atan2 which passes only one value to math.atan2(), when, to my knowledge, it should be taking (y, x) and passing both to math.atan2()

https://github.com/adobe-webplatform/Snap.svg/blob/master/src/svg.js

(Thanks for Snap -- lots of fun!)

### PR DESCRIPTION
**Pull request by mscuthbert**
Makes Snap.atan2 call math.atan2 with the proper number of arguments.  Fixes #507

### CODE REVIEW

File: src/svg.js [hunk 1]
```diff
@@ -379,11 +379,12 @@ Snap.atan = function (num) {
  [ method ]
  **
  * Equivalent to `Math.atan2()` only works with degrees, not radians.
- - num (number) value
+ - y (number) x-value
```
Comment (longsonr) [inline src/svg.js]: why is y the x-value and vice-versa?

Comment (mscuthbert) [inline src/svg.js]: Because I made a dumb mistake.  :-)  Fixed.

**Comment by DmitryBaranovskiy** [comment]
@mscuthbert You need to [Fill out the CLA](http://snapsvg.io/contributions/).

**Comment by mscuthbert** [comment]
Done!

**Comment by mscuthbert** [comment]
I think this is ready for a merge.

**Comment by orivej** [comment]
This was fixed in the `dev` branch by https://github.com/adobe-webplatform/Snap.svg/commit/2be525e0d64c8d587e4e27a74a3c1bfe30c85381

**Comment by DmitryBaranovskiy** [comment]
Already fixed

**Pull request closed without merging**